content/numpy/concepts/math-methods/terms/degrees/degrees.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: '.degrees()'
+Description: 'Converts radians to degrees.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+  - 'NumPy'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+In NumPy, the **`.degrees()`** function converts an angle measure from radians to degrees.
+
+## Syntax
+
+```pseudo
+numpy.degrees(x)
+
+numpy.degrees([x, x, x])
+```
+
+- `x`: The number or array of numbers in radians that must be converted.
```

Comment:
Please also describe the other parameters based on the above new syntax: `x`, `out` and `where`